.github/workflows/build.yml

```
@@ -257,6 +257,98 @@ jobs:
+  # Regenerates our own OpenCPN plugin catalog (one of alpha/beta/prod,
+  # whichever this push routed to) from the per-target metadata the jobs
+  # above already uploaded to Cloudsmith as "-metadata" raw packages, and
+  # pushes it to opencpn-radar-pi/plugins so OpenCPN's own "Update Plugin
+  # Catalog" -- pointed at CatalogChannel=custom, CatalogCustomURL=<ours> --
+  # sees a new build the moment this job finishes, not whenever the separate,
+  # hand-submitted metadata PR against OpenCPN/plugins happens to get
+  # reviewed. always() so one failed platform (which generate-catalog.sh
+  # already tolerates, skipping just that target) does not skip this too.
+  publish-catalog:
+    needs: [macos-universal, windows-msvc, linux, flatpak]
+    if: always() && env.CLOUDSMITH_API_KEY != '' && github.event_name != 'pull_request'
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)